shared-libs/cht-datasource/src/local/person.ts

```diff
@@ -92,4 +94,22 @@ export namespace v1 {
       )(limit, skip) as Page<Person.v1.Person>;
     };
   };
+
+  
+  /** @internal */
+  // eslint-disable-next-line max-len
+  export const createPerson = ({ medicDb, settings } : LocalDataContext) => async(qualifer: PersonQualifier) :Promise<Person.v1.Person> => {
+    if (hasField(qualifer, { name: '_rev', type: 'string', ensureTruthyValue: true })) {
+      throw new InvalidArgumentError('Cannot pass `_rev` when creating a person.');
+    }
+    
+    // This check can only be done when we have the contact_types from LocalDataContext.
+    if (!contactTypeUtils.isPerson(settings.getAll(), qualifer)) {
+      throw new InvalidArgumentError('Invalid contact type.');
+    }
+      
+    const createPerson = createDoc(medicDb);
+    const person = await createPerson(qualifer);
+    return person as Person.v1.Person;
```

Comment:
suggestion: this could be a one liner

```suggestion
    return await createPerson(qualifer) as Person.v1.Person;
```